src/utils/mongo.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose');
+
+const mongoPath = process.env.mongoPath;
```

Comment:
"mongoPath" should be renamed to "MONGO_PATH" to follow the variables that already exist in .env